src/cube_fit/linalg.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
-
+import scipy
+from scipy.spatial.transform import Rotation as R
```

Comment:
It is a bit dangerous to import this as `R` since you have a few variables further down that are also called `R`